string child_frame_id # the frame id of the child frame
string parent_frame_id
bool transform_valid
geometry_msgs/Transform transform #rotation and translation
